src/pages/workspaces/workspace/Dashboard/InfoRow.ts

```diff
@@ -0,0 +1,19 @@
+import { ReactNode } from 'react';
+import { dd, div, dt } from 'react-hyperscript-helpers';
+
+interface InfoRowProps {
+  title: string;
+  subtitle?: string;
+  children?: ReactNode;
```

Comment:
I think children are always passed (and subtitle never…).

Reply:
`subtitle` is used by `CloudInformation`
I made children optional largely for convention, matching the type signature of `PropsWithChildren`.